### ISSUE TITLE
NVDA fails to load in sum Applications

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#3921]
**Reported by netblue on 2014-02-25 07:38**
When unloading and reloading or loading  NVDA with shortcut control alt n within the open windows of serten applications. NVDA does not load but when navigating away from the application the shortcut key command works fine. So far this problem is with Firefox and VLC media player.
I am using Firefox 27 and VLC media player 2.0.8 on Windows 7 ultimate 32 bits.

**Comment by jcsteh** [comment]
We haven't seen any other reports of this nor any further info, so closing. Please reopen if the problem persists.
